ui/App/WalletScreen.svelte

```diff
@@ -90,13 +90,12 @@
   .container {
     min-width: var(--content-min-width);
     max-width: var(--content-max-width);
-    padding: var(--content-padding);
-    margin: 0 auto;
+    padding: 0 var(--content-padding);
+    margin: 2rem auto;
```

Comment:
As noted above this results in the screen being scrollable. I suggest
```suggestion
    padding: 2rem var(--content-padding);
    margin: 0 auto;
```